CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- Ability to configure the `MaxRamPercentage` and `MinRamPercentage` for the PlantUML process inside the container via `cesapp edit-conf`  (#61)
```

Comment:
Do you mean the nexus process?
In other PRs you noticed that the memory limit can be set via `cesapp`. Why did you omit it here?

Reply:
That's a typo. It should be Nexus, instead of PlantUML. The task included quite some C&P.

As for the memory limit keys, they were already integrated with a preceding release.